lib/apipie/resource_description.rb

```diff
@@ -106,6 +106,7 @@ def to_json(method_name = nil, lang = nil)
 
       {
         :doc_url => doc_url,
+        :resource_name => _id,
```

Comment:
I wonder if we shouldn't call it `:id`, as we already have the `:name` field, and it might be misleading

Reply:
Done